posthog/models/person/sql.py

```diff
@@ -337,12 +337,13 @@
 """
 
 DELETE_PERSON_BY_ID = """
-INSERT INTO person (id, created_at, team_id, properties, is_identified, _timestamp, _offset, is_deleted) SELECT %(id)s, %(created_at)s, %(team_id)s, %(properties)s, %(is_identified)s, %(_timestamp)s, 0, 1
+INSERT INTO person (id, created_at, team_id, properties, is_identified, _timestamp, version, _offset, is_deleted) SELECT %(id)s, %(created_at)s, %(team_id)s, %(properties)s, %(is_identified)s, %(_timestamp)s, %(version)s, 0, 1
 """
 
 DELETE_PERSON_EVENTS_BY_ID = """
-ALTER TABLE events DELETE
-WHERE distinct_id IN (
+ALTER TABLE {events_data_table}
+ON CLUSTER '{cluster}'
```

Comment:
This will blow up production as this means we'll run the deletion on backup nodes which don't have enough resources to process this query.

Reply:
I'm aware of this - but this code path is currently not called. 

Happy to delete all as I can see how this being here might cause us to make bad decisions, but the previous query was just plain wrong so I thought I should fix it. 

Might as well just remove all for now

Reply:
Let's rather remove this than invite disaster.